app/controllers/predicates/AuthoriseAsAgentOnly.scala

```diff
@@ -66,6 +68,9 @@ class AuthoriseAsAgentOnly @Inject()(enrolmentsAuthService: EnrolmentsAuthServic
       case _: AuthorisationException =>
         logger.warn("[AuthoriseAsAgentOnly][invokeBlock] - Authorisation Exception, rendering Technical Difficulties view")
         errorHandler.showInternalServerError
+      case error: UpstreamErrorResponse =>
+        logger.debug(s"[AuthoriseAsAgentOnly][invokeBlock] - Upstream error response received: ${error.message}")
```

Comment:
Could you change this and the same message in the other predicate to be warn level instead of debug. We'd want to be able to see these in Kibana